pilot/pkg/model/jwks_resolver_test.go

```diff
@@ -400,17 +400,21 @@ func TestJwtPubKeyLastRefreshedTime(t *testing.T) {
 }
 
 func TestJwtPubKeyRefreshWithNetworkError(t *testing.T) {
-	r := NewJwksResolver(JwtPubKeyEvictionDuration, time.Second /*RefreshInterval*/)
+	r := NewJwksResolver(JwtPubKeyEvictionDuration, time.Second /*RefreshInterval*/, testRetryInterval)
 	defer r.Close()
 
+	t.Logf("%v: %v", time.Now(), 0)
```

Comment:
the log here and below seems for local debugging?

Reply:
thanks, I'll remove that